cpu/stm32f4/ldscripts/stm32f413vg.ld

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (C) 2017 OTA keys S.A.
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @addtogroup      cpu_stm32f4
+ * @{
+ *
+ * @file
+ * @brief           Memory definitions for the STM32F413VG
+ *
+ * @author          Vincent Dupont <vincent@otakeys.com>
+ *
+ * @}
+ */
+
+MEMORY
+{
+    rom (rx)    : ORIGIN = 0x08000000, LENGTH = 1024K
+    ram (rwx)   : ORIGIN = 0x20000000, LENGTH = 256K
```

Comment:
The VG is a 1M flash variant.

But you're right about ram, I fixed stm32f413zh as well.

By the way, I'm wondering if we should not rename stm32f413xh and stm32f413xg as there is only one vendor header for all stm32f413xx and there is only two flash sizes (G = 1M, H = 1.5M).